addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_skins.py

```diff
@@ -55,11 +57,21 @@ def __filter_skin(blender_object, export_settings):
 
 
 def __gather_extensions(blender_object, export_settings):
-    return None
+    extension_wrapper = GLTF2ExtensionSkins(blender_object)
+    extensions = export_user_extensions(extension_wrapper, export_settings)
+    return extensions
 
 
 def __gather_extras(blender_object, export_settings):
-    return None
+    extensions = {}
+
+    # Handle custom user defined extensions
+    for user_extension_name, user_extension in export_settings['gltf_user_extensions'].items():
+        extension = user_extension.export(blender_object, export_settings)
+        if extension:
+            extensions[user_extension_name] = extension
+
+    return extensions
```

Comment:
Should be `return extensions or None` (not that I think it matters since empty dicts get erased at some point anyway, but for consistency...)

Reply:
This is an error. gether_extras should be unchanged.